nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ExecuteSQL.java

```diff
@@ -83,6 +83,8 @@
         @WritesAttribute(attribute = "executesql.query.fetchtime", description = "Duration of the result set fetch time in milliseconds"),
         @WritesAttribute(attribute = "executesql.resultset.index", description = "Assuming multiple result sets are returned, "
                 + "the zero based index of this result set."),
+        @WritesAttribute(attribute = "executesql.error.message", description = "If processing an incoming flow file causes "
+                + "an Exception, the flow file is routed to failure and this attribute is set to the exception message."),
```

Comment:
Can you capitalize "flow file" to "Flow File" in both processors?

Reply:
Sure, thanks for the correction.